src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -282,8 +283,15 @@ const ContentListTable = () => {
                     },
                   },
                   {
-                    isDisabled: actionTakingPlace || rowData?.status === 'Retrying',
-                    title: 'Trigger snapshot',
+                    isDisabled:
+                      actionTakingPlace || rowData?.status === 'Retrying' || !rowData.snapshot,
+                    title: rowData.snapshot ? (
+                      'Trigger snapshot'
+                    ) : (
+                      <Tooltip content='Snapshots disabled for this repository.' isVisible>
+                        <p>Trigger snapshot</p>
```

Comment:
not sure if this is the best way to add a tooltip here. thoughts?

Reply:
I think theres a custom component for this?
ConditionalTooltip

Reply:
Although you'd have to add a <Text> component or similar to ensure the isDisabled prop is being passed to something that can use it.